src/main/java/sirius/biz/model/ContactData.java

```diff
@@ -24,21 +22,24 @@
 import sirius.kernel.nls.NLS;
 import sirius.web.mails.Mails;
 
+import java.util.function.Consumer;
 import java.util.regex.Pattern;
 
 /**
  * Provides various contact information for a person or company which can be embedded into other entities or mixins.
+ * <p>
+ * Note that this class doesn't perform any save checks or validations at all. Any entity which contains this composite
+ * must decide which checks have to be performed and then either call the <tt>verifyXXX</tt> method within an
+ * {@link sirius.db.mixing.annotations.BeforeSave} handler or invoke <tt>validateXXX</tt> in an
+ * {@link sirius.db.mixing.annotations.OnValidate} method. Most probably these checks should be surrounded with
+ * a {@link sirius.db.mixing.BaseEntity#isChanged(Mapping...)} check to only validate or verify new values.
  */
 public class ContactData extends Composite {
 
     /**
-     * Validates a phone numner.
+     * Validates a phone number.
      */
-    public static final Pattern VALID_PHONE_NUMBER =
-            Pattern.compile("\\+?\\d+( \\d+)*( */( *\\d+)+)?( *\\-( *\\d+)+)?");
-
-    @Transient
-    private boolean validatePhoneNumbers;
+    public static final Pattern VALID_PHONE_NUMBER = Pattern.compile("\\+?\\d+( \\d+)*( */( *\\d+)+)?( *-( *\\d+)+)?");
```

Comment:
Is this format valid internationally?

Reply:
This regex isn't new / has not changed semantically. The \\- was a redundant escape which has been removed (as suggested by IntelliJ) - hopefully the regex was/is valid :)